metadata-helper/src/status.rs

```diff
@@ -0,0 +1,30 @@
+//! Status service.
+
+use crate::AppState;
+use actix_web::HttpResponse;
+
+/// Expose liveness status.
+///
+/// Status:
+///  * Live (200 code): The metrics endpoint has started running
+///  * Not Live (503 code): everything else.
+pub async fn serve_liveness(app_data: actix_web::web::Data<AppState>) -> HttpResponse {
+    if app_data.is_live() {
+        HttpResponse::Ok().finish()
+    } else {
+        HttpResponse::ServiceUnavailable().finish()
+    }
+}
+
+/// Expose readiness status.
+///
+/// Status:
+///  * Ready (200 code): the application has been initialized and is available to accept connections.
+///  * Not Ready (503 code): no JSON graph available yet.
```

Comment:
I'm fuzzy on what `app_data.is_ready` is actually  about, but the blocking issue for serving signatures out of this process is whether the local signature filesystem is present.  With init-container data sourcing, that's true from before this container comes up.  In a future where one of these containers is trying to poll graph-data image from the registry, it would be true after the first successful retrieval and unpacking.  Neither of those are directly about the JSON graph (that graph-builder is serving?).